source/iostreams.tex

```diff
@@ -14939,7 +14939,7 @@
 
 \pnum
 \begin{note} The current path as returned by many operating systems is a dangerous
-  global variable. It may be changed unexpectedly by a third-party or system
+  global variable. It may be changed unexpectedly by third-party or system
```

Comment:
For future reference, I'd generally prefer for unrelated fixes such as this and the "duration" -> "duration's" change below to be provided as a separate pull request.